openmetadata-spec/src/main/resources/json/schema/entity/services/pipelineService.json

```diff
@@ -188,6 +188,12 @@
     "domain" : {
       "description": "Domain the Pipeline service belongs to.",
       "$ref": "../../type/entityReference.json"
+    },
+    "sourceHash": {
+      "description": "Source hash of the entity",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 64
```

Comment:
do we need this for services? all we need is actual data that is coming from the sources such as tables/ topics/ dashboards etc..

Reply:
Makes sense I'll remove the field from service entities